packages/core/src/lib/passcode.test.ts

```diff
@@ -134,6 +134,7 @@ describe('sendPasscode', () => {
         name: {},
         logo: '',
         description: {},
+        readme: '',
```

Comment:
missing `'README.md'`?

Reply:
This mocks the return value of some methods, anything follows the type define should be good enough.